components/elm/cime_config/config_component.xml

```diff
@@ -217,34 +217,35 @@
-    <desc compset="_ELM%[^_]*FATES"    >clm4.5 FATES (Functionally Assembled Terr. Ecosystem Simulator): (experimental)</desc>
-    <desc compset="_ELM%[^_]*BGCDV"    >clm4.5 BGC (CN with vertically resolved soil BGC, based on Century with Methane) with dynamic veg</desc>
-    <desc compset="_ELM%[^_]*CRDCTCBC"   >alm1.0 C only, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNRDCTCBC"  >alm1.0 C-N, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNPRDCTCBC" >alm1.0 C-N-P, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNECACTCBC" >alm1.0 C-N, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNPECACTCBC">alm1.0 C-N-P, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM50"               >clm5.0 physics:</desc>
-    <desc compset="_ELM50%[^_]*SP"       >clm5.0 Satellite phenology:</desc>
-    <desc compset="_ELM50%[^_]*BGC"      >clm5.0 bgc (cn and methane):</desc>
+    <desc compset="_ELM"               >elm physics:</desc>
+    <desc compset="_ELM%[^_]*SP"       >elm Satellite phenology:</desc>
+    <desc compset="_ELM%[^_]*SPBC"     >elm Satellite phenology with black carbon deposition:</desc>
+    <desc compset="_ELM%[^_]*CN"       >elm cn:</desc>
+    <desc compset="_ELM%[^_]*CNDV"     >elm cn with dynamic vegetation:</desc>
+    <desc compset="_ELM%[^_]*BGC"      >elm bgc (cn and methane):</desc>
+    <desc compset="_ELM%[^_]*CROP"     >elm prognostic crop:</desc>
+    <desc compset="_ELM%[^_]*VIC"      >elm vic hydrology:</desc>
+    <desc compset="_ELM%[^_]*FATES"    >elm FATES (Functionally Assembled Terr. Ecosystem Simulator): (experimental)</desc>
+    <desc compset="_ELM%[^_]*BGCDV"    >elm BGC (CN with vertically resolved soil BGC, based on Century with Methane) with dynamic veg</desc>
+    <desc compset="_ELM%[^_]*CRDCTCBC"   >elm C only, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
+    <desc compset="_ELM%[^_]*CNRDCTCBC"  >elm C-N, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
+    <desc compset="_ELM%[^_]*CNPRDCTCBC" >elm C-N-P, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
+    <desc compset="_ELM%[^_]*CNECACTCBC" >elm C-N, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
+    <desc compset="_ELM%[^_]*CNPECACTCBC">elm C-N-P, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
+    <desc compset="_ELM50"               >elm physics:</desc>
```

Comment:
Are "ELM" and "ELM50" really distinct?

Reply:
> Are "ELM" and "ELM50" really distinct?

I'm not aware of anyone using the ELM50 compsets. I think this could get cleaned up in future revision.

Reply:
Actually this ELM50 block can be removed.  We have no compsets with "ELM50" in them.

Reply:
> Actually this ELM50 block can be removed. We have no compsets with "ELM50" in them.

@rljacob do you want this cleaned up now? There are a number of other CLM/ELM naming issues that this PR does not resolve. We were thinking of going back and doing a more complete cleanup after the v3 tag.

Reply:
I'll remove the descriptions in the push I'm about to make.  Other cleanup can wait.